src/screens/DiscoveryScreenV2/elements/Search.js

```diff
@@ -284,7 +284,7 @@ const styles = StyleSheet.create({
     height: dimen.normalizeDimen(33),
     paddingTop: 0,
     paddingBottom: 0,
-    color: COLORS.white2
+    color: COLORS.white
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the text has been changed from `COLORS.white2` to `COLORS.white`. Ensure that this change does not affect the visibility and contrast of the text against its background.
```diff
-    color: COLORS.white2
+    color: COLORS.white
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->